### ISSUE TITLE
[RFC] Deprecate the scssphp-import-once directive ?

### ISSUE THREAD

**Issue by stof** [scssphp/scssphp#337]
The `@scssphp-import-once` directive resolves imports like `@import`. But it then skips the processing if the file has already been imported (either through another `@scssphp-import-once` or through `@import`).
This is not standard feature of Sass.

However, the Sass module system provides something like that. `@use` will emit the styles only once.

Given that the implementation cost is low (the resolution logic is not different, and the list of included files is already tracked for the result object anyway), I'm not entirely sure we should deprecate that now. Maybe keeping until we have modules is better.

**Comment by stof** [comment]
@robocoder @Cerdic what do you think about this ?

**Comment by robocoder** [comment]
Deprecate.
